Presentation.Web/app/components/it-system/usage/tabs/it-system-usage-tab-GDPR.view.html

```diff
@@ -198,21 +200,62 @@ <h4>{{systemUsageName}}</h4>
             </div>
             <!-- 2nd column -->
             <div class="col-xs-6">
-
-            <div class="form-group" data-ng-show="usage.dataLevel == 2">
-                    <label class="control-label small">Følsomme persondata:</label>
+                <div class="form-group">
+                    <label class="control-label small">Hvilke typer data indeholder systemet</label>
                     <div class="well">
-                        <div class="checkbox" data-ng-repeat="r in sensitivePersonalData">
+                        <div class="checkbox">
                             <label>
-                                <input type="checkbox" ng-model="r.Checked"
-                                       data-ng-disabled="!hasWriteAccess"
-                                       ng-click="updateDataLevel(r.Id, r.Checked, 'SENSITIVEPERSONALDATA')">{{r.Name}}
+                                <input type="checkbox"
+                                       data-element-type="dataLevelTypeNoneCheckbox"
+                                       ng-change="dataLevelChange(sensitiveDataLevel.NONE)"
+                                       ng-model="usageViewModel.personalNoDataSelected"
+                                       data-ng-disabled="!hasWriteAccess">
+                                Ingen Persondata
```

Comment:
Disse tekster (oversættelser) står nu både i en mapper til OData og her.
En sensitivity level er en model med følgende attributter:
- **Value**: Numerisk værdi (fra enum)
- **TextValue**: "Tekstbaseret værdi" (fra enum) - bruges i dit odata vm
- **Text**: En oversættelse
- **EnumValue**: Værdien fra din enum **(har vi brug for den hvis vi har en model hvor de andre tre properties er der?**

Hvis du laver den generelle model og placerer den i en model, så kan du lave "hashmaps" ud fra den model der hvor du har brug for tekst-til-værdi og omvendt. På den måde har du kun 1 model dvs. ingen gentagelser af dine værdier. Det er det jeg gerne vil have dig til.

Hvis du er i tvivl om hvordan man laver hashmaps så sig til. Det er almindelige objekter som man smider properties ind på. Essensen er, at vi ikke skal gentage modellen alle mulige steder men gerne må **bruge** modellen.

Her er et hurtigt eksempel på et hashmap med udgangspunkt i colors fra excelhelperen (map metoden er ikke i koden i dag)

![image](https://user-images.githubusercontent.com/13116889/76929282-07bed300-68e4-11ea-9781-b6a71c19a478.png)

Reply:
Når man har sådan et map i hånden så kan man tilgå det på nøglen  dvs. map["red"] --> "Rød" og for dem der ikke findes map["bla"] -> undefined

Reply:
Der hvor du så har faste udfaldsrum (ikke kendo hvor det er dynamish mapping) kan du så bare gå direkte på din model, så hvis vi forestiller os at du skulle sætte teksten i din viewmodel et eller andet sted, så ville det se således ud:

`$enEllerAndenVm.sensitivityLevelNone = Kitos.Models.Generic.SensitivityLevels.all.none`
og vi view ville du jo så tilgå:
`ng-change="dataLevelChange(enEllerAndenVm.sensitivityLevelNone.Value)"`
og
`{{enEllerAndenVm.sensitivityLevelNone.Text}}`